operators/bosh-restore-operator/BoshRestoreService.js

```diff
@@ -509,6 +510,12 @@ class BoshRestoreService extends BaseDirectorService {
 
   async processPostStart(resourceOptions) { 
     await this.runErrand(resourceOptions, 'postStartErrand'); 
+    if (this.service.pitr === true) {
+      await this.reScheduleBackup({
```

Comment:
Do we want to fail the restore if reScheduling fails?
In RestoreService this part is put under tap, https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/operators/restore-operator/RestoreService.js#L111

Reply:
Good point! I will make the changes. @abh1kg I think this makes sense, right?

Reply:
makes sense to go with @amitmalav on this

Reply:
@amitmalav can you check this now? I guess Jayesh has taken care of it in the latest fix. Please approve if it looks fine to you.